Makefile

```
@@ -1,12 +1,12 @@
 # These "find" commands match text.xml and *.svg files, respectively,
 # but only after excluding the .git directory from the search for
 # performance and overall sanity reasons.
-HTMLS := $(subst text.xml,index.html,\
-  $(shell find ./ -name .git -prune -o \( -type f -name 'text.xml' -print \)))
-IMAGES := $(patsubst %.svg,%.png,\
-  $(shell find ./ -name .git -prune -o \( -type f -name '*.svg' -print \)))
+XMLS := $(shell find . -name .git -prune -o -type f -name 'text.xml' -print)
+SVGS := $(shell find . -name .git -prune -o -type f -name '*.svg' -print)
+HTMLS := $(subst text.xml,index.html,$(XMLS))
+PNGS := $(patsubst %.svg,%.png,$(SVGS))
```

I'm skeptical about the change from `IMAGES` to `PNGS`. Can we do:

```suggestion
PNGS := $(patsubst %.svg,%.png,$(SVGS))
IMAGES := $(PNGS)
```

---

I have reverted the variable name to IMAGES.

---

Bikeshed only: I thought `PNGS` was fine, because the "png" extension is mentioned explicitly and it's consistent with `XMLS` and `HTMLS`. The place where you would want `IMAGES` is in the default target, because you want to build all images there. Something like

```
IMAGES = $(PNGS) $(JPGS)
all: ... $(IMAGES)
```

But obviously not a big deal.